package.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 SRC_FOLDER=dist/src/main/java/org/json
+MODITECT_FOLDER=dist/src/moditect
 WORKING_DIR=`pwd`
 
 # Get the original sources from Douglas Crockfords GitHub repo
 mkdir -p $SRC_FOLDER
+#mkdir -p $MODITECT_FOLDER
+# Copy the module-info to the dist folder to build
+#cp src/moditect/module-info.java $MODITECT_FOLDER/module-info.java
+#echo "Copied the module info file"
```

Comment:
What about these 4 lines? Are they neccessary, not neccessary or optional? Or are there conditions when they should be enabled?